### PR TITLE
Fixed #716

### LINKED ISSUE #716: Should we add a filter box to news, map catalogue and other pages which do not have it?
ODC team suggested to offer a filtering box to users on:

- Map catalogue: https://pp.opendevelopmentcambodia.net/layers
- News archive: https://pp.opendevelopmentcambodia.net/news

Although the users should be encouraged to use the main search function, it would be positive to give the users the possibility to filter out results presented on these pages. Possible filters might be:

- Textual search
- Taxonomy

@prustar @ChandaraKhun @DBishton @gimmemochi Please add your comments here.

### PR DESCRIPTION
**Pull request by acorbi**
# DO NOT MERGE THIS PR, THE CURRENT CANDIDATE FOR MERGING IS on https://github.com/OpenDevelopmentMekong/wp-odm_theme/pull/736

### CODE REVIEW

**Comment by acorbi** [comment]
@Huyeng  Please test this PR in PP and provide comments. So far this code change is adding a filter box ( textual search, topic and date) on:
- announcements archive: https://pp.opendevelopmentmekong.net/announcements
- map catalogue: https://pp.opendevelopmentmekong.net/layers
- maps archive: https://pp.opendevelopmentmekong.net/maps/
- news archive: https://pp.opendevelopmentmekong.net/news
- profiles archive: https://pp.opendevelopmentmekong.net/profiles
- site-updates archives: https://pp.opendevelopmentmekong.net/updates
- stories archives: https://pp.opendevelopmentmekong.net/stories

screenshot from 2017-01-30 13-15-12

screenshot from 2017-01-30 13-16-49

@DBishton @gimmemochi @prustar any comments?

File: bower.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "datatables": "~1.10.12",
     "chosen": "~1.5.1",
     "fontawesome": "~4.6.3",
-    "leaflet": "^0.7.7"
+    "leaflet": "^0.7.7",
+    "moment": "~2.17.1"
```
Comment (Huyeng) [inline bower.json]: What is moment using for?

Comment (acorbi) [inline bower.json]: @Huyeng moment is a javscript library to work with dates, I strongly recomend it to you: http://momentjs.com/

it is used here https://github.com/OpenDevelopmentMekong/wp-odm_theme/blob/3e6eb94b849112ef3df64674b9d037b46cbd7f86/inc/advanced-navigation.php#L213

**Comment by acorbi** [comment]
Closing this one since it is now invalid

File: bower.json (hunk 1 above)
Comment (Huyeng) [inline bower.json]: Yeah, I knew. However, i would like to ask where will the script be used on the site?

**Pull request closed without merging**